README.md

```diff
@@ -45,25 +44,88 @@ Create an input configuration object and add a redux-inputs reducer to your redu
         inputs: createInputsReducer(inputConfig)
     });
 
-Note that `inputs` is the default mount point for redux-inputs state. You can
- configure this in your `inputConfig` object.
+### Step 4 - Connect
 
-    inputConfig = {
-        _form: {
-            reduxMountPoint: 'loginModal.userInputs'
-        }
-        ... other inputs config ...
-    }
+    import { connectWithInputs } from 'redux-inputs';
+    
+    YourForm = connectWithInputs(inputConfig)()(YourForm);
+    
+Connect your form with `connectWithInputs`, which takes the `inputConfig` and creates a function that has the same 
+interface as react-redux's `connect`. This will pass down these additional props to your component, in addition to any 
+other redux state specified:
 
-In this example your state would look like this:
+#### Props
 
-    {
-        loginModal: {
-            userInputs: { ... }
-        }
+- `inputProps` *{Object}* contains props for each input in the inputConfig
+
+- `inputActions` *{Object}*
```

Comment:
All of the react-redux connect params are still optional in the connect function. 

The whole point of using this connect function over react-redux's connect is to pass down these additional props. There are only 3 of them, and the names for them can be configured (needs documentation) if there are name-collision issues. 

redux-form did the manual action mapping thing in some of the much earlier versions, but it was a lot more configuration for very little benefit, and they've just passed down everything for at least the last 3 major versions.